spec/enumerable_spec.rb

```diff
@@ -1,28 +1,22 @@
 # frozen_string_literal: true
 
-require './lib/enumerable_1.rb'
-require './lib/enumerable_2.rb'
-require './lib/multiply_els.rb'
-
 describe '#my_each' do
-  before(:each) do
-    @arr = []
-    %w[a b c].my_each { |x| @arr.push(x) }
-  end
   it 'fill an array with another array values' do
-    expect(@arr).to eql(%w[a b c])
+    arr = []
+    %w[a b c].my_each { |x| arr.push(x) }
```

Comment:
- [ ] Instead of using `before_each` to define a variable for the test cases, students should [Use `let` instead](https://github.com/rubocop-hq/rspec-style-guide#let-blocks). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/before_each.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
- [ ]  Use `let` for all your variables. You could do this:
```ruby
let(:arr){ ["a", "b", "c"] }
```